src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Simplification/SymbolAnnotation.cs

```diff
@@ -17,6 +17,9 @@ internal class SymbolAnnotation
         public static SyntaxAnnotation Create(ISymbol symbol)
             => new(Kind, DocumentationCommentId.CreateReferenceId(symbol));
 
+        internal static SyntaxAnnotation Create(string referenceId)
+            => new(Kind, referenceId);
```

Comment:
what's this for?

Reply:
Previously, we was having "System.Console" as string (via `typeof(Console).FullName`), we then get a compilation and get the INamedTypeSymbol for System.Console, just to convert it to a string back via `DocumentationCommentId.CreateReferenceId(symbol)`.

This simplifies the code by adding an overload that takes the reference id directly.